Gemfile

```diff
@@ -3,8 +3,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6.0.2', '>= 6.0.2.1'
+gem 'rails', '~> 6.1'
```

Comment:
in discussion earlier, we said pin to patch level as rails isn't great at avoiding glitches in backwards compatibility

Reply:
Ahhh.

Reply:
Yes, Rails does not follow semver.